build.gradle.kts

```diff
@@ -18,28 +18,28 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-data-jpa")
 
   implementation("org.apache.commons:commons-text:1.13.0")
-  implementation("io.swagger:swagger-annotations:1.6.14")
-  implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.7.0")
+  implementation("io.swagger:swagger-annotations:1.6.15")
+  implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.8.3")
   implementation("org.springframework:spring-jms")
   implementation("com.google.code.gson:gson:2.11.0")
   implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:5.2.2")
-  implementation("org.apache.commons:commons-csv:1.12.0")
-  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:2.9.0")
+  implementation("org.apache.commons:commons-csv:1.13.0")
+  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:2.12.0")
```

Comment:
this should really be `2.9` which matches https://github.com/microsoft/ApplicationInsights-Java/blob/3.6.2/dependencyManagement/build.gradle.kts#L16